src/kit.ts

```diff
@@ -973,6 +973,18 @@ export async function scanForKits(opt?: KitScanOptions) {
   });
 }
 
+// Rescan if the kits versions (extension context state var versus value defined for this release) don't match.
+export async function scanForKitsIfNeeded(context: vscode.ExtensionContext) : Promise<void> {
+  const kitsVersionSaved = context.workspaceState.get<number>('kitsVersionSaved');
```

Comment:
this should be `context.globalState`.

Reply:
good catch I just created new PR for this quick fix.